### PR TITLE
feat: "dump catalog" debug CLI

### PR DESCRIPTION
**Pull request by crepererum**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -176,7 +179,6 @@ fn main() -> Result<(), std::io::Error> {
 
         match config.command {
             Command::Database(config) => {
-                let _tracing_guard = handle_init_logs(init_simple_logs(log_verbose_count));
```
Comment (carols10cents) [inline src/main.rs:179]: Is removing log initialization from the `database` commands intentional? This change didn't look related to dumping the catalog to me...

**Review by carols10cents — APPROVED** [review]
One teeny tiny question, otherwise LGTM!

File: src/main.rs (hunk 1 above)
Comment (NGA-TRAN) [reply src/main.rs:179]: +1

**Review by NGA-TRAN — APPROVED** [review]
LGTM

File: src/main.rs (hunk 1 above)
Comment (crepererum) [reply src/main.rs]: fixed

**Pull request merged**